aspnet/tutorials/your-first-mac-aspnet.rst

```diff
@@ -1,53 +1,88 @@
 Your First ASP.NET Core Application on a Mac Using Visual Studio Code
 =====================================================================
 
-By `Daniel Roth`_, `Steve Smith`_ and `Rick Anderson`_
+By `Daniel Roth`_, `Steve Smith`_, `Rick Anderson`_ and `Shayne Boyer`_
 
 This article will show you how to write your first ASP.NET Core application on a Mac.
 
 .. contents:: Sections:
   :local:
   :depth: 1
 
+
 Setting Up Your Development Environment
 ---------------------------------------
 
-To setup your development machine download and install `.NET Core`_ and `Visual Studio Code`_ with the `C# extension <https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp>`__.
+To setup your development machine download and install `.NET Core`_ and `Visual Studio Code`_ with the `C# extension <https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp>`__. Node.js and npm is also required, if not already installed visit `nodejs.org <https://nodejs.org/en/download/package-manager/#osx>`_. 
 
 Scaffolding Applications Using Yeoman
 -------------------------------------
 
-Follow the instruction in :doc:`/client-side/yeoman` to create an ASP.NET Core project.
+We will be using ``yo aspnet`` to generate the **Web Application Basic** template, you may follow the full instructions in :doc:`/client-side/yeoman` to create an ASP.NET Core project which show an **Empty Web** for reference.
```

Comment:
How about just this:

We will use `` :doc:`Yeoman </client-side/yeoman>` `` to generate an initial ASP.NET Core project.